Documentation/articles/getting_started/1_creating_a_new_project_vs.md

```diff
@@ -1,19 +1,75 @@
 # Creating a Project with Visual Studio
 
-Start up Visual Studio and select **New Project...** in the upper left corner.
+MonoGame is available on Windows for development using the instructions below to get started with MonoGame 3.8.
+
+> Note, MonoGame 3.8 projects are no longer compatible with earlier versions of MonoGame.  If you wish to work on or build older MonoGame projects, then you will still need to install [MonoGame 3.7.1](https://www.monogame.net/downloads/) or earlier to open them.
+
+## Prerequisites
+
+You will need a copy of [Visual Studio 2019](https://www.monogame.net/downloads/) or later installed (any edition, including Community) before installing MonoGame, with the following components (depending on your target platform):
+
+![Visual Studio optional components](~/images/getting_started/1_installer_vs_components.png)
+
+* .Net Desktop Development - For Windows GL / DX platforms
```

Comment:
This isn't necessary, unless you want to target .NET Framework with WindowsDX to support old Windows versions or to ship without an embedded .NET runtime. The .NET Core payload is required.

Reply:
Maybe I'll reword, but without this component you also would never be able to open older 3.7 packages.  But I grant you for the netcore/netstandard projects it may not be required, I'll have to test.